pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -123,6 +127,41 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 	return events
 }
 
+// TODO: kube-schduler should unregister the CSINode events once all the volume plugins has completed their CSI migration.
+func (pl *VolumeBinding) isSchedulableAfterCSINodeChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
+	oldCSINode, modifiedCSINode, err := util.As[*storagev1.CSINode](oldObj, newObj)
+	if err != nil {
+		return framework.Queue, err
+	}
+
+	logger = klog.LoggerWithValues(
+		logger,
+		"Pod", klog.KObj(pod),
+		"CSINode", klog.KObj(modifiedCSINode),
+	)
+
+	oldAnn := oldCSINode.ObjectMeta.Annotations[v1.MigratedPluginsAnnotationKey]
+	modifiedAnn := modifiedCSINode.ObjectMeta.Annotations[v1.MigratedPluginsAnnotationKey]
+
+	oldAnnSet := sets.New[string]()
+	if oldAnn != "" {
+		oldAnnSet = sets.New(strings.Split(oldAnn, ",")...)
+	}
+
+	modifiedAnnSet := sets.New[string]()
+	if modifiedAnn != "" {
+		modifiedAnnSet = sets.New(strings.Split(modifiedAnn, ",")...)
+	}
+
+	if !oldAnnSet.Equal(modifiedAnnSet) {
+		logger.V(5).Info("CSINode was created or updated and this may make the Pod rejected by VolumeBinding plugin in the previous scheduling cycle schedulable", "pod", klog.KObj(pod), "CSINode", klog.KObj(modifiedCSINode))
```

Comment:
You can eliminate `"pod", klog.KObj(pod), "CSINode", klog.KObj(modifiedCSINode)` from logs because you already add them at L137-L141.

Reply:
I eliminate some unnecessary args from log.
[d0e758b](https://github.com/kubernetes/kubernetes/pull/125097/commits/d0e758b1342f4955884850cb3fdc3f62e81d8195)